releasetasks/release_configs/dev_mozilla-beta_fennec_rc_graph_2.yml

```diff
@@ -0,0 +1,47 @@
+# we don't want to do any en-US, l10n, or parter based tasks for graph 2
+platforms: {}
+release_platforms: []
+final_verify_platforms:
+  - "android-4-0-armv7-api15"
+  - "android-x86"
+uptake_monitoring_platforms:
+  - "android-api-15"
+  - "android-x86"
+l10n_release_platforms: []
+partner_repacks_platforms: []
+eme_free_repacks_platforms: []
+sha1_repacks_platforms: []
+
+updates_builder_enabled: false
+source_enabled: false
+bouncer_enabled: false
+# funsize
+updates_enabled: false
+checksums_enabled: false
+push_to_candidates_enabled: false
+update_verify_enabled: false
+push_to_releases_enabled: true
+# rc graph 2 only needs final_verify_channels
+channels:
+publish_to_balrog_channels:
+    - "beta-dev"
```

Comment:
@rail shouldn't we use here `beta`? The configs are switching gears to use the staging Balrog instance anyway. I couldn't find any `beta-dev` channel there. I'm not sure how this works in Firefox 😕